node/tasks.py

```diff
@@ -80,7 +80,10 @@ def send_app_txs(app_name: str) -> None:
         )
     except Exception:
         zlogger.exception("An unexpected error occurred:")
-        zdb.add_missed_txs(app_name=app_name, txs=initialized_txs)
+        if len(initialized_txs) != 0:
+            zdb.add_missed_txs(app_name=app_name, txs=initialized_txs)
+            return
+        zdb.is_sequencer_down = True
```

Comment:
when does this flag set to `False`?
I think after each successful sending tx to sequencer it should be set to `False`

Reply:
It has been set to `False` before the node wants to send initialized transactions or heartbeat.